cmd/tfimport/main.go

```diff
@@ -70,6 +73,25 @@ func run() error {
 		importRn = &runner.Default{}
 	}
 
+	retry := true
+	for retry {
```

Comment:
It think this should be capped at max 5 retries

Reply:
Why do you suggest that? This isn't a "retry until it works" loop, it's more of a "retry as long as we're making some progress".

Reply:
Ah ok, I see if all resources error on retry then it will stop. That's fine then